lib/ddtrace/contrib/redis/ext.rb

```diff
@@ -9,7 +9,9 @@ module Ext
         ENV_ANALYTICS_ENABLED_OLD = 'DD_REDIS_ANALYTICS_ENABLED'.freeze
         ENV_ANALYTICS_SAMPLE_RATE = 'DD_TRACE_REDIS_ANALYTICS_SAMPLE_RATE'.freeze
         ENV_ANALYTICS_SAMPLE_RATE_OLD = 'DD_REDIS_ANALYTICS_SAMPLE_RATE'.freeze
+        ENV_COMMAND_ARGS = 'DD_REDIS_COMMAND_ARGS'.freeze
         METRIC_PIPELINE_LEN = 'redis.pipeline_length'.freeze
+        PIPELINE_RESOURCE = 'PIPELINE'.freeze
```

Comment:
I think this can be removed now.
```suggestion
        PIPELINE_RESOURCE = 'PIPELINE'.freeze
```